tomviz/DockerExecutor.cxx

```diff
@@ -228,6 +227,7 @@ void DockerPipelineExecutor::containerError(int containerExitCode)
                            QString("Docker logs failed with: %1\n\n%2")
                              .arg(exitCode)
                              .arg(logsInvocation->stdErr()));
+              logsInvocation->deleteLater();
               return;
```

Comment:
I added a `deleteLater()` here rather than removing the return statement because I was not sure, if the docker logs invocation failed, whether we wanted to remove the container or not, which is done below.